README.md

```diff
@@ -1,70 +1,113 @@
 
-In the project directory, you can run:
+## Built With
 
-### `npm start`
+- [x] Languages: HTML, Sass and JavaScript
+- [x] Linters: Stylelint and ESLint
+- [x] Front-end library: React & Redux
+- [x] Code Editor: VS Code
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## Live Demo 🔗
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+[Demo link](https://frabjous-bienenstitch-c3048c.netlify.app/)
 
-### `npm test`
+## Loom Video 🔗
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+[Loom Video](https://www.loom.com/share/f0cfce89d0714aaf905a1a28c08c47cf)
 
-### `npm run build`
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+## Getting Started
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+To get a local copy up and running follow these simple steps.
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+### Prerequisites
 
-### `npm run eject`
+- [x] A web browser like Google Chrome.
+- [x] A code editor like Visual Studio Code with Git and Node.js.
+- [x] Check a app on your Mobile.
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+You can check if Git is installed by running the following command in the terminal.
+```
+$ git --version
+```
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+Likewise for Node.js and npm for package installation.
+```
+$ node --version && npm --version
+```
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+### Setup
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+Clone the repository using the GitHub link provided below.
 
-## Learn More
+### Install
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+In the terminal, go to your file directory and run this command.
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+```
+$ git clone git@github.com:mavericks-db/math-magicians.git
```

Comment:
- [ ] [OPTIONAL] Please, consider fixing the current project name in this section - it seems, `math-magicians` came from previous one 👍